src/root/components/ThreeWForm/index.tsx

```diff
@@ -599,6 +600,15 @@ function ThreeWForm(props: Props) {
                 value={value.status ? statuses[value.status] : undefined}
               />
             </div>
+            <div>
+              <Switch
+                  label="Annual Reporting"
+                  name="is_annual_report"
+                  value={true}
+                  onChange={true}
+                  invertedLogic
```

Comment:
Just a note: `intervtedLogic` prop will invert the behaviour, i.e. Switch will be off when the value is true and on if the value is false. You may not need it in this situation

Reply:
Thank you – it was just a yesterday's quick copy of Switch to the needed place.